Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Creating-Saving-and-Publishing-Content/index.md

```diff
@@ -11,45 +10,54 @@ Firstly you will need to select the parent page for your new page. This could be
+
+![This shows an arrow pointing at The save button.](images/The-save-button.png)
 
 ### Save and Publish
 
 The ***Save and publish*** button is ideal for when you want to publish a previously saved page to the live website, or to publish a page without previewing it. This will save and publish the page to your live website.
 
-### Publish
+![This shows an arrow pointing at Save and publish.](images/Save-and-publish.png)
+
+### Publish with descendants
+
+Clicking the arrow next to the ***Save and publish*** button will present you with two options(three if you have published before).
+Here you can pick ***Publish with descendants***, This will allow you to publish the current page and all content that is linkrd to this page.
+You will get the option to include drafts, this means that you also include all unpublished content items for thsi page and all linked pages.
 
-Clicking the actions menu in the top right of the page or right clicking on the name of the page you wish to publish from the context menu will reveal the slide out menu. On this menu you will see an option to ***Publish***. You can now choose to ***Publish*** all sub-pages excluding any with unpublished changes. You can also select to include all unpublished sub-pages pages too. This is especially useful if you have many pages you wish to publish at the same time.
+![This shows an arrow pointing at Publish with descendants.](images/Publish-with-descendants.png)
 
-### Scheduled Publishing
+### Scheduled Publishing and Unpublishing
 
-Scheduled publishing is ideal when you need a page to go live at a specific date and time when it is not convenient to do so manually. To schedule publishing you need to navigate to the ***Properties*** tab and click on the Calendar icon within the Publish at field which will then open up a ***Calendar***. Select the date and then click the ***Clock*** icon at the bottom of the calendar to select the time you wish for the page to be published. After you have done this, click anywhere outside of the date and time picker pop-up to close it. Now all you need to do is save your changes by clicking the arrow on the ***Save and publish*** button and click ***Save***.
-Umbraco will automatically publish the page at the date the time you chose.
+Scheduled publishing is ideal when you need a page to go live at a specific date and time when it is not convenient to do so manually. To schedule publishing/unpublishing you need to navigate to the arrow next to the ***Save and publish*** button and select the ***Schedule*** button then a box will pop up, that ask for a date for the schedualed publishing or unpublishing. Select the date and then click the ***Clock*** at the bottom of the calendar to select the time you wish for the page to be published/unpublished. After you have done this, click anywhere outside of the date and time picker pop-up to close it. Now all you need to do is save your changes by clicking The ***Schedule*** button.
```

Comment:
This section is very long. Wouldn't it make sense to split the bottom part into a list instead?
At least the steps on how to publish / unpublish.. What do you think? 🤔

```suggestion
Scheduled publishing is ideal when you need a page to go live at a specific date and time, and it is not convenient to do so manually. 

To schedule publishing or unpublishing a page you need to navigate to the arrow next to the ***Save and publish*** button and select the ***Schedule*** option. Then a box will pop up, that asks for a date for the scheduled publishing or unpublishing. Select the date and then click the ***Clock*** at the bottom of the calendar to select the time you wish for the page to be published/unpublished. After you have done this, click anywhere outside of the date and time picker pop-up to close it. Now all you need to do is save your changes by clicking The ***Schedule*** button.
```